--- install/Makefile	2020-02-28 11:20:12.229672273 +0800
+++ openssl-1.1.1d/Makefile	2020-02-28 11:23:23.387086624 +0800
@@ -123,8 +123,8 @@
 # Variables starting with CNF_ are common variables for all product types
 
 CNF_CPPFLAGS=-DNDEBUG
-CNF_CFLAGS=-pthread -m64
-CNF_CXXFLAGS=-std=c++11 -pthread -m64
+CNF_CFLAGS=-pthread
+CNF_CXXFLAGS=-std=c++11 -pthread
 CNF_LDFLAGS=
 CNF_EX_LIBS=-ldl -pthread
 
